### ISSUE TITLE
Ringtone settings gets disabled after rotating the device

### ISSUE THREAD

**Issue by rezazarchi** [thunderbird/thunderbird-android#7468]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.713

### Where did you get the app from?

F-Droid

### Android version

14

### Device model

Samsung Galaxy S23 Ultra

### Steps to reproduce

1. Go to your account settings
2. Go to Notifications
3. Select "New mail ringtone
4. In the opened window, rotate the device
5. press the back button
6. scroll down to see the "New mail ringtone" and other two related options

### Expected behavior

We should see them enabled and we could click on them again.

### Actual behavior

"**New mail ringtone**" and "**Vibration**" and "**Notification light**" are disabled

### Video

https://github.com/thunderbird/thunderbird-android/assets/8654398/08b5be61-96c5-442c-bce5-1153fc592624

**Comment by rezazarchi** [comment]
@cketti 
Why did you disable the preference [here](https://github.com/thunderbird/thunderbird-android/blob/cba9ca31aa6bdb8911a2787afc145c27cf366bec/app/ui/legacy/src/main/java/com/fsck/k9/ui/settings/account/AccountSettingsFragment.kt#L211)?

```java
...
findPreference<NotificationSoundPreference>(PREFERENCE_NOTIFICATION_SOUND)?.let { preference ->
                notificationSoundPreference = preference
                preference.isEnabled = false
            }
...
```

**Comment by cketti** [comment]
At the time `initializeNotifications()` is called, `Account.notificationSettings` might contain outdated information. The current values are read from the `NotificationChannel` in `updateNotificationPreferences()`.

**Comment by rezazarchi** [comment]
> At the time `initializeNotifications()` is called, `Account.notificationSettings` might contain outdated information. The current values are read from the `NotificationChannel` in `updateNotificationPreferences()`.

Thanks for your answer but i didn't get how they are related to that line? How we can avoid this issue when returning to the AccountSettingsFragment from the rotated Ringtone manager activity ?

**Comment by cketti** [comment]
@rezazarchi: Can you test the fix in #7469?

**Comment by rezazarchi** [comment]
> @rezazarchi: Can you test the fix in #7469?

I checked this pull request and tested it on my Galaxy phone and verified that the issue has been resolved. Thank you.
